x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/apis/get_rules_muted_alerts.ts

```diff
@@ -23,9 +24,12 @@ export const getMutedAlerts = async (
   params: { ruleIds: string[] },
   signal?: AbortSignal
 ) => {
+  const filterNode = nodeBuilder.or(
+    params.ruleIds.map((id) => nodeBuilder.is('alert.id', `alert:${id}`))
+  );
   return http.post<FindRulesResponse>(INTERNAL_FIND_RULES_URL, {
     body: JSON.stringify({
-      rule_type_ids: params.ruleIds,
+      filter: JSON.stringify(filterNode),
```

Comment:
The snapshot in the tests will fail. Could you also add a test where there are multiple `ruleIds`? The current one only tests `ruleIds: ['foo']`.